Integrations/MicrosoftGraphMail/MicrosoftGraphMail.py

```diff
@@ -626,6 +1289,134 @@ def get_email_as_eml_command(client: MsGraphClient, args):
+            'flag': args.get('flag', 'notFlagged'),
+            'importance': args.get('importance', 'Low'),
+            'internet_message_headers': argToList(args.get('headers')),
+            'attach_ids': argToList(args.get('attachIDs')),
+            'attach_names': argToList(args.get('attachNames')),
+            'attach_cids': argToList((args.get('attachCIDs'))),
+            'manual_attachments': args.get('manualAttachObj', [])
+        }
+
+    elif command == 'reply-to':
+        return {
+            'to_recipients': argToList(args.get('to')),
+            'message_id': args.get('ID', ''),
+            'comment': args.get('body')
+        }
+
+    return args
+
+
+def create_draft_command(client: MsGraphClient, args):
+    """
+    Creates draft message in user's mailbox, in draft folder.
+    """
+    prepared_args = prepare_args('create-draft', args)
+    email = args.get('from')
+    suffix_endpoint = f'/users/{email}/messages'
+    draft = client.build_message(**prepared_args)
+
+    created_draft = client.ms_client.http_request('POST', suffix_endpoint, json_data=draft)
+
+    parsed_draft = client.parse_item_as_dict(created_draft)
+    headers = ['ID', 'From', 'Sender', 'To', 'Subject', 'Body', 'BodyType', 'Cc', 'Bcc', 'Headers', 'Importance',
+               'MessageID', 'ConversationID', 'CreatedTime', 'SentTime', 'ReceivedTime', 'ModifiedTime', 'IsDraft',
+               'IsRead']
+    human_readable = tableToMarkdown(f'Created draft with id: {parsed_draft.get("ID", "")}',
+                                     parsed_draft, headers=headers)
+    entry_context = {CONTEXT_DRAFT_PATH: parsed_draft}
+
+    return_outputs(human_readable, entry_context, created_draft)
+
+
+def build_recipients_human_readable(message_content):
+    to_recipients = []
+    cc_recipients = []
+    bcc_recipients = []
+
+    for recipients_dict in message_content.get('toRecipients', {}):
+        to_recipients.append(recipients_dict.get('emailAddress', {}).get('address'))
+
+    for recipients_dict in message_content.get('ccRecipients', {}):
+        cc_recipients.append(recipients_dict.get('emailAddress', {}).get('address'))
+
+    for recipients_dict in message_content.get('bccRecipients', {}):
+        bcc_recipients.append(recipients_dict.get('emailAddress', {}).get('address'))
+
+    return to_recipients, cc_recipients, bcc_recipients
+
+
+def send_email_command(client: MsGraphClient, args):
+    """
+    Sends email from user's mailbox, the sent message will appear in Sent Items folder
+    """
+    prepared_args = prepare_args('send-mail', args)
+    email = args.get('from', client._mailbox_to_fetch)
+    suffix_endpoint = f'/users/{email}/sendMail'
+    message_content = MsGraphClient.build_message(**prepared_args)
+    client.ms_client.http_request('POST', suffix_endpoint, json_data={'message': message_content}, resp_type="text")
+
+    message_content.pop('attachments', None)
+    message_content.pop('internet_message_headers', None)
```

Comment:
why are we removing it, we should have the names in the context

Reply:
Agreed on keeping it